src/components/Dialogs/TransactionDialog/TransactionDialog.tsx

```diff
@@ -45,12 +45,11 @@ const TransactionDialog: React.FC<TransactionDialogProps> = ({ status, onClose,
       ? TRANSACTION_STEPS_DETAILS[status]
       : null
 
-  const canCancel = status === ExtrinsicStatus.ProcessingAssets || ExtrinsicStatus.Unsigned
+  const canCancel = status === ExtrinsicStatus.ProcessingAssets || status === ExtrinsicStatus.Unsigned
```

Comment:
😅

Reply:
haha I wish it actually worked like that honestly

Reply:
for real, me too